src/libcore/mem.rs

```diff
@@ -177,15 +177,57 @@ pub fn forget<T>(t: T) {
 
 /// Returns the size of a type in bytes.
 ///
-/// More specifically, this is the offset in bytes between successive
-/// items of the same type, including alignment padding.
+/// More specifically, this is the offset in bytes between successive elements
+/// in an array with that item type including alignment padding. Thus, for any
+/// type `T` and length `n`, [T; n] will have a size of `n * size_of<T>()`. 
+///
+/// In general, the size of a type is not stable across compilations, but
+/// specific types such as primitives are.
+///
+/// The following table gives the size for primitives.
+///
+/// Type | size_of<Type>()
```

Comment:
`size_of::<Type>()`

Reply:
As durka wrote, this would be clearer as `size_of::<Type>()` instead of `size_of<Type>()`.

Reply:
Also I believe these need to be in backticks or escaped with a backslash. Otherwise markdown will treat it as a \<Type> HTML tag.

```
`Type` | `size_of::<Type>()`
```

```
Type | size_of::\<Type>()
```